settings.gradle

```diff
@@ -34,7 +34,6 @@ include(':stdlib')
 //include(':ballerina-crypto')
```

Comment:
We should probably remove what's commented out too, right? Maybe we can do that in a separate PR.